config/settings/base.py

```diff
@@ -204,12 +204,13 @@
 INSTALLED_APPS += [
     "django_celery_beat",
 ]
+
+CELERYD_CONCURRENCY = env("CELERYD_CONCURRENCY", default=500)
+CELERYD_POOL = env("CELERYD_POOL", default="gevent")
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-broker_url
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", default="django://")
 # https://docs.celeryproject.org/en/stable/userguide/optimizing.html#broker-connection-pools
-CELERY_BROKER_POOL_LIMIT = env(
-    "CELERY_BROKER_POOL_LIMIT", default=env("CELERYD_CONCURRENCY", default=500)
-)
+CELERY_BROKER_POOL_LIMIT = env("CELERY_BROKER_POOL_LIMIT", default=CELERYD_CONCURRENCY)
```

Comment:
Should this be `BROKER_POOL_LIMIT`? https://docs.celeryproject.org/en/stable/userguide/configuration.html?highlight=CELERYD_CONCURRENCY#broker-pool-limit

Reply:
Celery on Python uses the `CELERY` namespace for every configuration variable